gen/gen.go

```diff
@@ -94,32 +113,32 @@ func Struct(s string, dst io.Writer, gc Config) error {
 	return err
 }
 
-func generateFrom(schema avro.Schema, acc *data) string {
+func (gc Config) generateFrom(schema avro.Schema, acc *data) string {
```

Comment:
I get the idea to attach it to something, but it seems weird that it is the `Config` struct. Perhaps we need a `generator` struct that takes the `Config` and everything attaches to that?

Reply:
```go
type Generator struct {
  Config Config
}

func (g Generator) Struct(schema string, dst io.Writer) error {
  // ...
}
```

Call site as:

```go
schema := "..."
c := gen.Config{...}
generator := gen.Generator{Config: c}
err := generator.Struct(schema)
...
```

Something like this?

Reply:
I was thinking more like:
```go
type generator struct {
    Config Config
}

func Struct(schema string, dst io.Writer, cfg Config) error {
  gen := generator{Config: Config}

  // ...
}
```

Reply:
Sure, I had originally made it like that.